masterlist.yaml

```diff
@@ -6769,6 +6769,47 @@ plugins:
       # version: 1.06a
       - crc: 0x20AA5984
         util: 'SSEEdit v3.2.2'
+  - name: 'Immersive Movement.esp'
+    group: *lowPriorityGroup
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/15649/' ]
+    msg:
+      - <<: *patchIncluded
+        subs: [ 'Mortal Enemies' ]
+        condition: 'active("Mortal Enemies.esp") and not active("Immersive Movement - Mortal Enemies Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Smilodon' ]
+        condition: 'active("Smilodon - Combat of Skyrim.esp") and not active("Immersive Movement - Smilodon Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Wildcat' ]
+        condition: 'active("Wildcat - Combat of Skyrim.esp") and not active("Immersive Movement - Wildcat Patch.esp")'
+    clean:
+      # version: 1.3.2
+      - crc: 0xB91C4616
+        util: 'SSEEdit v4.0.2'
+  - name: 'Immersive Movement - Mortal Enemies Patch.esp'
+    after:
+      - 'Immersive Movement.esp'
+      - 'Mortal Enemies.esp'
```

Comment:
An after is fine for Mortal Enemies but all Mortal enemy variants should be added.

`Mortal Enemies.esp`
`Mortal Enemies - No RunWalk Changes.esp`
`Mortal Enemies - Rival Remix.esp`
`Mortal Enemies - Rival Remix - NoRunWalkChanges.esp`
`Mortal Enemies - 2.0 Test Version.esp`

Reply:
Done. Also added the variants to the condition.